README.md

```diff
@@ -37,14 +37,41 @@ dotnet run --project TKeazirian.HTTPServer
 
 This project uses xUnit.net for testing. Visit [this][5] website for more information and how to use xUnit.net.
 
+This project also uses Acceptance tests written in Cucumber Code found [here][9]. You can read more about Cucumber Code [here][10].
+
 ### To Run the Tests:
 
 ```shell
 dotnet test
 ```
 
+### Acceptance Tests:
+**Setup**
```

Comment:
When making a choice between [emphasis vs headers](https://cirosantilli.com/markdown-style-guide/#emphasis-vs-headers) in markdown, choose headers.

```suggestion
#### Setup
```